### ISSUE TITLE
has a bug in utils.py in BottleneckResnetBlock

### ISSUE THREAD

**Issue by ghzhangnj** [MIPT-Oulu/pytorch_bn_fusion#2]
BottleneckResnetBlock‘s forwad function lack of  'out += residual'  in resnet structure

**Comment by lext** [comment]
@ghzhangnj Could you provide a quantitative comparison between the resnet50 and resnet50 after the batch norm fusion? 

The included modules were adapted from the original PyTorch implementation.

**Comment by xuyuan** [comment]
https://github.com/MIPT-Oulu/pytorch_bn_fusion/blob/2f97390a0dd200f73725afaf3d84ed19780175dd/utils.py#L135
should be
```python
    if self.se_module is not None:
         out += self.se_module(out)
    out += residual
```

**Comment by lext** [comment]
@xuyuan could you make a PR with the fix?
